src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -200,6 +201,34 @@ public void testStructuredLogging(CapturedOutput capturedOutput) {
     assertThat((String) readJson(event4, "$.foo.bar")).isEqualTo("baz");
   }
 
+  @Test
+  public void testAlertLogging(CapturedOutput capturedOutput) {
+    ResponseEntity<String> response =
+        testRestTemplate.getForEntity("/testAlertLogging", String.class);
+    assertThat(response.getStatusCode().value()).isEqualTo(200);
+
+    String[] lines = capturedOutput.getAll().split("\n");
+    String event1 = null;
+    String event2 = null;
+    for (String line : lines) {
+      String message = readJson(line, "$.message");
+      if (message != null && message.contains("test alert message")) {
+        event1 = line;
+      } else if (message != null && message.contains("test structured object alert message")) {
+        event2 = line;
+      }
+    }
+
+    assertThat(event1).isNotNull();
+    assertThat((String) readJson(event1, "$.severity")).isEqualTo("ERROR");
+    assertTrue((Boolean) readJson(event1, "$." + LoggingUtils.ALERT_KEY));
+    assertThat(event2).isNotNull();
+    assertThat((String) readJson(event2, "$.severity")).isEqualTo("ERROR");
+    assertThat((String) readJson(event2, "$.pojo.name")).isEqualTo("asdf");
```

Comment:
same with these

Reply:
refactored `readJson` to read objects properly